bindings/node/.npmignore

```diff
@@ -2,7 +2,6 @@ target
 Cargo.lock
 .cargo
 .github
-npm
```

Comment:
revert because the presence of this file may not auto-include that an adjacent gitignore.